data/human/transport missions.txt

```diff
@@ -3209,3 +3209,76 @@ mission "Earth Retirement"
 			label reject
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+
+
+mission "Spacediving professionals"
+	name "Spacediving at <destination>"
+	description "Drop skydivers from the edge of space. They are carrying cooled spacesuits, durable parachutes and oxygen bottles to ensure the survival of their party. Payment is <payment>."
+	passengers 5 10
+	repeat
+	job
+	source
+		government "Republic" "Free Worlds" "Syndicate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
+	to offer
+		and
+			credits >= 1000000
+			random > 96
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 8000
```

Comment:
Only 8000?

Reply:
I reduced the distance. What amount would you feel is appropriate for this type of job?

Reply:
Typically payment is at least partially based on distance. Is there any reason to deliberately deviate from that convention?

Reply:
Allowing people to skydive from your ship carries higher risks than just transporting people from one location to another. I was thinking of just a fixed amount of higher payment regardless of the distance.
(And another reason is that I needed to hardcode the payment value for missions where the payment is actually not paid out.)

Reply:
The payment calculation is as follows: `base + multiplier * (jumps + 1) * (10 * (# of passengers) + (tons of cargo))`.
Given the max possible distance of 7 jumps, the max possible number of passengers of 10, and using a base of 0 and default multiplier of 150, we're looking at `0 + 150 * (7 + 1) * (10 * 10 + 0) = 120k` if this wasn't a fixed payment job. So just a fixed payment of 8k is hardly worth it given what a similarly sized normal passenger mission would offer.

```
mission "Skydiving Test"
	job
	repeat
	description "Head to <destination> with <bunks> passengers for <payment>."
	passengers 10
	destination
		distance 7
	on complete
		payment
```
![image](https://user-images.githubusercontent.com/17688683/123171507-c5623b00-d430-11eb-892c-a12abc07f165.png)

Even a bare minimum of a 1 distance destination with only 5 passengers would result in 15k credits. 
So this payment should definitely be increased. Given that some of the time that you take this mission you won't receive any pay or a reduced pay, I think a payment closer to the maximum possible under normal circumstances would be appropriate, possible even above the maximum.

Reply:
Agreed. I increased the payment to the maximum calculated here (and adjusted the fine to match).